docs/howtos/collaborative-development.md

```diff
@@ -42,15 +42,15 @@ store or a repository like Zenodo or Figshare.
 
 ### Permissively License Your Code and Data
 
-(adapted from [O3 guidelines](https://osf.io/vuzt3/)))
+(adapted from [O3 guidelines](https://osf.io/vuzt3/))
 
 Using recognizable, permissive licenses (e.g., CC0, CC BY) encourages contribution and ensures content longevity.
```

Comment:
Add MIT to the beginnig (used in linkml-cookiecutter)? 
or be more explicit
`(e.g. for code: MIT or Apache 2.0; for models or documentation: CC0 or CC BY)`